tests/integration/tools/test_freehand_draw_tool.py

```diff
@@ -0,0 +1,163 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest ; pytest
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# Standard library imports
+import time
+
+# External imports
+
+# Bokeh imports
+from bokeh.layouts import column
+from bokeh.models import ColumnDataSource, CustomAction, CustomJS, Plot, Range1d, MultiLine, FreehandDrawTool, Div
+from bokeh._testing.util.compare import cds_data_almost_equal
+from bokeh._testing.util.selenium import RECORD
+
+#-----------------------------------------------------------------------------
+# Tests
+#-----------------------------------------------------------------------------
+
+pytest_plugins = (
+    "bokeh._testing.plugins.bokeh",
+)
+
+def _make_plot(num_objects=0):
+    source = ColumnDataSource(dict(xs=[], ys=[]))
+    plot = Plot(plot_height=400, plot_width=400, x_range=Range1d(0, 3), y_range=Range1d(0, 3), min_border=0)
+    renderer = plot.add_glyph(source, MultiLine(xs='xs', ys='ys'))
+    tool = FreehandDrawTool(num_objects=num_objects, renderers=[renderer])
+    plot.add_tools(tool)
+    plot.toolbar.active_multi = tool
+    code = RECORD("xs", "source.data.xs") + RECORD("ys", "source.data.ys")
+    plot.add_tools(CustomAction(callback=CustomJS(args=dict(source=source), code=code)))
+    plot.toolbar_sticky = False
+    return plot
+
+def _make_server_plot(expected, num_objects=0):
+    def modify_doc(doc):
+        source = ColumnDataSource(dict(xs=[], ys=[]))
+        plot = Plot(plot_height=400, plot_width=400, x_range=Range1d(0, 3), y_range=Range1d(0, 3), min_border=0)
+        renderer = plot.add_glyph(source, MultiLine(xs='xs', ys='ys'))
+        tool = FreehandDrawTool(num_objects=num_objects, renderers=[renderer])
+        plot.add_tools(tool)
+        plot.toolbar.active_multi = tool
+        div = Div(text='False')
+        def cb(attr, old, new):
+            if cds_data_almost_equal(new, expected):
+                div.text = 'True'
+        source.on_change('data', cb)
+        code = RECORD("matches", "div.text")
+        plot.add_tools(CustomAction(callback=CustomJS(args=dict(div=div), code=code)))
+        doc.add_root(column(plot, div))
+    return modify_doc
+
+
+@pytest.mark.integration
+@pytest.mark.selenium
+class Test_FreehandDrawTool(object):
+
+    def test_selected_by_default(self, single_plot_page):
+        plot = _make_plot()
+
+        page = single_plot_page(plot)
+
+        target = 'bk-tool-icon-freehand-draw'
+
+        button = page.driver.find_element_by_class_name(target)
+        assert 'active' in button.get_attribute('class')
+
+        assert page.has_no_console_errors()
+
+    def test_can_be_deselected_and_selected(self, single_plot_page):
+        plot = _make_plot()
+
+        page = single_plot_page(plot)
+
+        target = 'bk-tool-icon-freehand-draw'
```

Comment:
use `get_toolbar_button` for this 

https://github.com/bokeh/bokeh/blob/master/bokeh/_testing/plugins/bokeh.py#L199-L200

Reply:
Will do.